.mergify.yml

```diff
@@ -39,3 +39,24 @@ pull_request_rules:
             is updated. Please revert or drop all changes to the `web-client` submodule from this PR and perform any
             required frontend work by opening and merging a PR against
             [cryostat-web](https://github.com/cryostatio/cryostat-web).
+
+  - name: request reviews from the reviewers team
+    conditions:
+      - label=review-requested
+    actions:
+      request_reviews:
+        teams:
+          - "@reviewers"
```

Comment:
`@reviewers` did you mean `cryostatio/reviewers` ?

Reply:
/request_review

Reply:
Good catch: https://docs.mergify.com/actions/request_reviews/#flexible-reviewers-assignment